### PR TITLE
Distinctions and Overlaps

### PR DESCRIPTION
**Pull request by susanatx**
-struck age-impairments from definition of accessibility.
-Included the word Inclusion in the first sentence of that section
-moved "age" to the list of issues in inclusion.

I'm hesitant to continue to include age as a subset of disability. Disabilities can, and are likely, to come with age but being aged in and of itself is not a disability.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ status: editors-draft
   <h2 id="terms">Distinctions and Overlaps</h2>
   <dl>
     <dt id="accessibility"><strong>Accessibility</strong></dt>
-      <dd><em>Accessibility</em> addresses discriminatory aspects related to equivalent user experience for people with disabilities, including people with age-related impairments. For the web, accessibility means that people with disabilities can perceive, understand, navigate, and interact with websites and tools, and that they can contribute equally without barriers. For more information, see the <a href="https://www.w3.org/standards/webdesign/accessibility">Accessibility introduction</a>.</dd>
+      <dd><em>Accessibility</em> addresses discriminatory aspects related to equivalent user experience for people with disabilities. For the web, accessibility means that people with disabilities can perceive, understand, navigate, and interact with websites and tools, and that they can contribute equally without barriers. For more information, see the <a href="https://www.w3.org/standards/webdesign/accessibility">Accessibility introduction</a>.</dd>
```
Comment (nitedog) [inline index.md:22]: @susanatx where do you read "age as a subset of disability"? In the past, leaving out "age-related impairments" as a type of disability, led to people thinking that accessibility work is less relevant to designing for older people. I think this is reinforcing your point, that "Disabilities can, and are likely, to come with age but being aged in and of itself is not a disability".

File: index.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ status: editors-draft
...
     <dt id="inclusion"><strong>Inclusion</strong></dt>
-      <dd><em>Inclusive design</em>, <em>universal design</em>, and <em>design for all</em> involves designing products, such as websites, to be usable by everyone to the greatest extent possible. Inclusion addresses a broad range of issues including access to and quality of hardware, software, and Internet connectivity; computer literacy and skills; economic situation; education; geographic location; and language &mdash; as well as age and disability.</dd>
+    <dd><em>Inclusion</em> or <em>Inclusive design</em>, <em>universal design</em>, and <em>design for all</em> involves designing products, such as websites, to be usable by everyone to the greatest extent possible. Inclusion addresses a broad range of issues including access to and quality of hardware, software, and Internet connectivity; computer literacy and skills; economic situation; education; geographic location; age; and language &mdash; as well as disability.</dd>
```
Comment (nitedog) [inline index.md:26]: @susanatx what is the rationale for including "Inclusion" again in the text (it is already the heading)?

Comment (shawna-slh) [reply index.md:26]: -1 for adding "Inclusion" in the text. It's used in the heading as a shortcut for all those things listed and doesn't fit well added to the sentence.

no strong opinion on moving age in the last sentence

File: index.md (hunk 1 above)
Comment (susanatx) [reply index.md:22]: "Age as a subset" was my wording. Besides pwd, designing for accessibility is important for aged, temporarily disabled, those in disabling environments (e.g. when using mobile in certain situations), no? How do these the differ? (Other than getting old being a drag.)

File: index.md (hunk 2 above)
Comment (susanatx) [reply index.md:26]: What Shawn said. :)

File: index.md (hunk 1 above)
Comment (shawna-slh) [reply index.md:22]: related comment: https://github.com/w3c/wai-inclusion/pull/15#issuecomment-263951921

**Pull request closed without merging**